powerhome/playbook
# [PLAY-705] Changing the Phone Number Input returned value format

#### Breaking Changes

No

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-705)

#### How to test this

Go to the Phone Number Input kit page and check the new value returned from the onChange prop.

#### Checklist:
